docs/source/guides/importpilotmodel.rst

```diff
@@ -0,0 +1,224 @@
+.. _importpilotmodel:
+
+Pilot Model Importing
+=====
+
+Tools:
+^^^^^
+
+.. _Blender: https://www.blender.org/
+.. _Blender Source Tools: http://steamreview.org/BlenderSourceTools/
+.. _Titanfall VPK Tools: https://retryy.gitbook.io/tf2/Wiki/Tools/vpk-tools
+.. _Crowbar: https://steamcommunity.com/groups/CrowbarTool
+.. _Legion+: https://github.com/r-ex/LegionPlus
+.. _RePak: https://github.com/r-ex/RePak
+.. _mdlshit: https://github.com/headassbtw/mdlshit-binaries
+.. _Text Editors: https://retryy.gitbook.io/tf2/Wiki/Tools/general-pc-tools#text-editing
+
+Download each tool. "Titanfall VPK Tool" is the vpk tool used. Made by cra0.
+
+======================== =========================== ===================
+   `Legion+`_              `RePak`_                    `mdlshit`_
+   `Text Editors`_         `Blender`_                    `Crowbar`_
+`Titanfall VPK Tools`_   `Blender Source Tools`_           
+======================== =========================== ===================
```

Comment:
Why are these in a table?

Reply:
Compact. I thought the 9 paragraphs for each links was excessive.

Reply:
Alright. It'll be changed then.